Formula/gtksourceview5.rb

```diff
@@ -0,0 +1,47 @@
+class Gtksourceview5 < Formula
```

Comment:
Does this name follow our standards? https://docs.brew.sh/Versions#acceptable-versioned-formulae

Reply:
well there are already `gtksourceview`, `gtksourceview3` and `gtksourceview4` formulas so it made sense to me name it `gtksourceview5`.

Reply:
Actually, it looks like this formula should be the one called `gtksourceview4` (as it's the one that depends on `gtk4`). At least that's the only way I can make sense of the naming convention for `gtksourceview` and `gtksourceview3`.

We can call it `gtksourceview4@5`? (Just kidding; let's not.)